front/app/modules/commercial/analytics/admin/containers/Visitors/VisitorsOverview.tsx

```diff
@@ -46,7 +46,7 @@ const VisitorsOverview = ({ uniqueVisitorDataDate }: Props) => {
     setProjectId(value);
   };
 
-  if (!uniqueVisitorDataDate) {
+  if (!uniqueVisitorDataDate.isValid()) {
```

Comment:
I think it would be cleaner to make the `uniqueVisitorDataDate` prop `Moment | undefined`, and then in the parent (`front/app/modules/commercial/analytics/admin/containers/Visitors/index.tsx`) only create the `uniqueVisitorDataDate` if `typeof countData.first_dimension_date_first_action_date === 'string'`. Something like

```ts
// front/app/modules/commercial/analytics/admin/containers/Visitors/index.tsx
const firstDate = countData.first_dimension_date_first_action_date;

const uniqueVisitorDataDate = typeof firstDate === 'string'
  ? moment(firstDate)
  : undefined;
```
```

Reply:
Makes sense. Done.